parser/remote/grpc/pom.xml

```diff
@@ -16,35 +16,60 @@
 -->
 
 <project xmlns="http://maven.apache.org/POM/4.0.0"
-xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-<parent>
-  <groupId>org.sqlflow</groupId>
-  <artifactId>remote-parsers</artifactId>
-  <version>0.1-SNAPSHOT</version>
-  <relativePath>../pom.xml</relativePath>
-</parent>
-<modelVersion>4.0.0</modelVersion>
-<artifactId>parser-grpc</artifactId>
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.sqlflow</groupId>
+    <artifactId>remote-parsers</artifactId>
+    <version>0.1-SNAPSHOT</version>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+  <artifactId>parser-grpc</artifactId>
 
-<build>
-  <extensions>
-    <extension>
-      <groupId>kr.motd.maven</groupId>
-      <artifactId>os-maven-plugin</artifactId>
-    </extension>
-  </extensions>
-  <plugins>
-    <plugin>
-      <artifactId>maven-jar-plugin</artifactId>
-    </plugin>
-    <plugin>
-      <artifactId>maven-dependency-plugin</artifactId>
-    </plugin>
-    <plugin>
-      <groupId>org.xolstice.maven.plugins</groupId>
-      <artifactId>protobuf-maven-plugin</artifactId>
-    </plugin>
-  </plugins>
-</build>
+  <dependencies>
+    <dependency>
+      <groupId>io.grpc</groupId>
+      <artifactId>grpc-netty-shaded</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.grpc</groupId>
+      <artifactId>grpc-protobuf</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.grpc</groupId>
+      <artifactId>grpc-stub</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>commons-cli</groupId>
+      <artifactId>commons-cli</artifactId>
```

Comment:
move all version to parent pom, so there is one central place to manage all dependencies’ version.

Reply:
I see. Will do.